scripts/statistics.js

```diff
@@ -1,23 +1,34 @@
 'use strict';
 const bcd = require('..');
+const { RANGE_PREFIX_LT } = require('../utils.js');
```

Comment:
Should you call this `RANGE_PREFIX_LTE`? 'LT' means 'less than', but the actual symbol is 'less than or equal to'. It could be a pain if someday we actually need  LT in addition to LTE.